frontend/packages/operator-lifecycle-manager/src/components/descriptors/spec/resource-requirements.tsx

```diff
@@ -79,13 +79,12 @@ export const ResourceRequirements: React.FC<ResourceRequirementsProps> = (props)
 export const ResourceRequirementsModal = withHandlePromise(
   (props: ResourceRequirementsModalProps) => {
     const { obj, path, type, model } = props;
-    const [cpu, setCPU] = React.useState<string>(_.get(obj.spec, `${path}.${type}.cpu`, ''));
-    const [memory, setMemory] = React.useState<string>(
-      _.get(obj.spec, `${path}.${type}.memory`, ''),
-    );
+    const [cpu, setCPU] = React.useState<string>(_.get(obj.spec, `${path}.${type}.cpu`));
```

Comment:
Is this a candidate for the typescript syntax instead? `obj.spec?.path?.type?.cpu`